gambit/README.md

```diff
@@ -0,0 +1,35 @@
+# Gambit MIMIC Integration
+
+## Overview
+
+Gambit MIMIC is a dynamic, real-time, high-performance SNMP, Netflow, sFlow Simulator. All Netflow and sFlow exports are recorded by the simulator and replayed with flexible customization toward Datadog agent(s) and presented with all existing available metrics of Datadog, whether built-in or Datadog marketplace integrated.
+
+## Setup
+
+### Installation
+
+Follow the configuration instructions below.
+
+### Configuration
+
+1. If you are already using Datadog and have the Datadog Agent installed, configure Gambit MIMIC agent to export Netflow and sFlow toward the Datadog Agent.
```

Comment:
How does the user "configure Gambit MIMIC agent to export Netflow and sFlow toward the Datadog Agent"?